src/component/component/index.js

```diff
@@ -137,19 +137,31 @@ export class Component extends BaseComponent {
         // Register all of the drivers for instantiating components. The model used is -- there's a standard javascript
         // way of rendering a component, then each other technology (e.g. react) needs to hook into that interface.
         // This makes us a little more pluggable and loosely coupled.
-
+        this.driverResults = {};
```

Comment:
Should we then change: ```registerDrivers``` method to have:
```
if (glob) 
{
       this[driverName] = this.driver(driverName, glob);
}
```
instead of:
```
if (glob) 
{
       this.driver(driverName, glob);
}
```